packages/autodiscovery/src/Discovery.php

```diff
@@ -17,33 +17,33 @@ final class Discovery
     /**
      * @var AutodiscoveryFinder
      */
-    private $fileSystem;
+    private $autodiscoveryFinder;
 
     /**
      * @param string[] $packageDirectories
      */
     public function __construct(string $projectDirectory, array $packageDirectories = [])
     {
-        $this->fileSystem = new AutodiscoveryFinder($projectDirectory, $packageDirectories);
+        $this->autodiscoveryFinder = new AutodiscoveryFinder($projectDirectory, $packageDirectories);
```

Comment:
@dobryy Hey, just wanted to share fruits of your work :).

I've just added the "naming" Rector set to Symplify packages, and it's fixing many wrong places like this one. I'm happy I don't have to care about this little details anymore :hugs: